src/facade/dragonfly_connection.h

```diff
@@ -61,6 +61,12 @@ class Connection : public util::Connection {
 
   virtual void SendMsgVecAsync(const PubMessage& pub_msg, util::fibers_ext::BlockingCounter bc);
 
+  // Please note, this accept the message by value, since we really want to
+  // create a new copy here, so that we would not need to "worry" about memory
+  // management, we are assuming that we would not have many copy for this, and that
+  // we would not need in this way to sync on the lifetime of the message
+  virtual void SendMonitorMsg(std::string monitor_msg);
```

Comment:
why is it virtual? where we override it?

Reply:
Just to match the SendMsgVecAsync - I think this is required for UT, so I need UT for this!